### ISSUE TITLE
Search isn't working atm

### ISSUE THREAD

**Issue by joeykrug** [AugurProject/augur#341]
Just does nothing upon typing something in and hitting enter, then won't allow me to backspace/erase what i typed in

**Comment by tinybike** [comment]
What browser/OS is this on?  (Is this still happening?)  You're not supposed to even have to hit enter -- it should filter the results as you type.  It works ok for me on Chrome/Ubuntu.

**Comment by joeykrug** [comment]
Let's me search once then the not able to delete characters and search not working at all happens. iOS Safari and Windows 10 on Firefox

**Comment by joeykrug** [comment]
Lets*

**Comment by tinybike** [comment]
It works fine for me on Firefox + Ubuntu.  @thinkloop can you try and reproduce this on Windows + Firefox?

**Comment by thinkloop** [comment]
I tested it on FF/windows and Chrome/windows - works well. Is it still not
working @joeykrug? This was likely due to an uncaught js error. If you
experience it again could you check your console for any messages.

**Comment by joeykrug** [comment]
Ok, it's working on Windows again.. trying mobile

**Comment by joeykrug** [comment]
Actually, just got it to fail on windows too!

**Comment by joeykrug** [comment]
`TypeError: tourMarketKey is undefined`

```[129]</render/<()
 app.js:53372
map()
 self-hosted:240
render()
 app.js:53371
[824]</</ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext()
 app.js:128803
[824]</</ReactCompositeComponentMixin._renderValidatedComponent()
 app.js:128823
[863]</</ReactPerf.measure/wrapper()
 app.js:135320
[824]</</ReactCompositeComponentMixin._updateRenderedComponent()
 app.js:128776
[824]</</ReactCompositeComponentMixin._performComponentUpdate()
 app.js:128760
[824]</</ReactCompositeComponentMixin.updateComponent()
 app.js:128689
[863]</</ReactPerf.measure/wrapper()
 app.js:135320
[824]</</ReactCompositeComponentMixin.performUpdateIfNecessary()
 app.js:128637
[868]</ReactReconciler.performUpdateIfNecessary()
 app.js:136013
runBatchedUpdates()
 app.js:136716
[892]</</Mixin.perform()
 app.js:138614
[892]</</Mixin.perform()
 app.js:138614
[875]</</<.perform()
 app.js:136673
[875]</</flushBatchedUpdates()
 app.js:136734
[863]</</ReactPerf.measure/wrapper()
 app.js:135320
[892]</</Mixin.closeAll()
 app.js:138680
[892]</</Mixin.perform()
 app.js:138627
[839]</ReactDefaultBatchingStrategy.batchedUpdates()
 app.js:131337
enqueueUpdate()
 app.js:136763
enqueueUpdate()
 app.js:136351
[874]</</ReactUpdateQueue.enqueueSetState()
 app.js:136517
[821]</</ReactComponent.prototype.setState()
 app.js:128060
handlePageChanged()
 app.js:53243
[129]</MarketsPage<.debounceSearchInput<()
 app.js:53261
complete()
 app.js:99157
delayed()
 app.js:99167
 app.js:53372:17
[ethrpc] primary node: "https://eth4.augur.net" app.js:146359:9
[filter] contracts: 0x84c9426dfabdd67f5971e01aa670108b3d484897 app.js:48184:11
contracts filter: TypeError: tourMarketKey is undefined
Stack trace:
[129]</render/<@https://app.augur.net/app.js:53372:17
render@https://app.augur.net/app.js:53371:185
[824]</</ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext@https://app.augur.net/app.js:128803:29
[824]</</ReactCompositeComponentMixin._renderValidatedComponent@https://app.augur.net/app.js:128823:27
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[824]</</ReactCompositeComponentMixin._updateRenderedComponent@https://app.augur.net/app.js:128776:31
[824]</</ReactCompositeComponentMixin._performComponentUpdate@https://app.augur.net/app.js:128760:5
[824]</</ReactCompositeComponentMixin.updateComponent@https://app.augur.net/app.js:128689:7
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[824]</</ReactCompositeComponentMixin.receiveComponent@https://app.augur.net/app.js:128621:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[824]</</ReactCompositeComponentMixin._updateRenderedComponent@https://app.augur.net/app.js:128778:7
[824]</</ReactCompositeComponentMixin._performComponentUpdate@https://app.augur.net/app.js:128760:5
[824]</</ReactCompositeComponentMixin.updateComponent@https://app.augur.net/app.js:128689:7
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[824]</</ReactCompositeComponentMixin.receiveComponent@https://app.augur.net/app.js:128621:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[824]</</ReactCompositeComponentMixin._updateRenderedComponent@https://app.augur.net/app.js:128778:7
[824]</</ReactCompositeComponentMixin._performComponentUpdate@https://app.augur.net/app.js:128760:5
[824]</</ReactCompositeComponentMixin.updateComponent@https://app.augur.net/app.js:128689:7
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[824]</</ReactCompositeComponentMixin.receiveComponent@https://app.augur.net/app.js:128621:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[818]</</ReactChildReconciler.updateChildren@https://app.augur.net/app.js:126997:9
[858]</</ReactMultiChild.Mixin._reconcilerUpdateChildren@https://app.augur.net/app.js:134626:1
[858]</</ReactMultiChild.Mixin._updateChildren@https://app.augur.net/app.js:134761:26
[858]</</ReactMultiChild.Mixin.updateChildren@https://app.augur.net/app.js:134736:9
[828]</</ReactDOMComponent.Mixin._updateDOMChildren@https://app.augur.net/app.js:129961:7
[828]</</ReactDOMComponent.Mixin.updateComponent@https://app.augur.net/app.js:129790:5
[828]</</ReactDOMComponent.Mixin.receiveComponent@https://app.augur.net/app.js:129735:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[818]</</ReactChildReconciler.updateChildren@https://app.augur.net/app.js:126997:9
[858]</</ReactMultiChild.Mixin._reconcilerUpdateChildren@https://app.augur.net/app.js:134626:1
[858]</</ReactMultiChild.Mixin._updateChildren@https://app.augur.net/app.js:134761:26
[858]</</ReactMultiChild.Mixin.updateChildren@https://app.augur.net/app.js:134736:9
[828]</</ReactDOMComponent.Mixin._updateDOMChildren@https://app.augur.net/app.js:129961:7
[828]</</ReactDOMComponent.Mixin.updateComponent@https://app.augur.net/app.js:129790:5
[828]</</ReactDOMComponent.Mixin.receiveComponent@https://app.augur.net/app.js:129735:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[818]</</ReactChildReconciler.updateChildren@https://app.augur.net/app.js:126997:9
[858]</</ReactMultiChild.Mixin._reconcilerUpdateChildren@https://app.augur.net/app.js:134626:1
[858]</</ReactMultiChild.Mixin._updateChildren@https://app.augur.net/app.js:134761:26
[858]</</ReactMultiChild.Mixin.updateChildren@https://app.augur.net/app.js:134736:9
[828]</</ReactDOMComponent.Mixin._updateDOMChildren@https://app.augur.net/app.js:129961:7
[828]</</ReactDOMComponent.Mixin.updateComponent@https://app.augur.net/app.js:129790:5
[828]</</ReactDOMComponent.Mixin.receiveComponent@https://app.augur.net/app.js:129735:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[818]</</ReactChildReconciler.updateChildren@https://app.augur.net/app.js:126997:9
[858]</</ReactMultiChild.Mixin._reconcilerUpdateChildren@https://app.augur.net/app.js:134626:1
[858]</</ReactMultiChild.Mixin._updateChildren@https://app.augur.net/app.js:134761:26
[858]</</ReactMultiChild.Mixin.updateChildren@https://app.augur.net/app.js:134736:9
[828]</</ReactDOMComponent.Mixin._updateDOMChildren@https://app.augur.net/app.js:129961:7
[828]</</ReactDOMComponent.Mixin.updateComponent@https://app.augur.net/app.js:129790:5
[828]</</ReactDOMComponent.Mixin.receiveComponent@https://app.augur.net/app.js:129735:5
[868]</ReactReconciler.receiveComponent@https://app.augur.net/app.js:135998:5
[824]</</ReactCompositeComponentMixin._updateRenderedComponent@https://app.augur.net/app.js:128778:7
[824]</</ReactCompositeComponentMixin._performComponentUpdate@https://app.augur.net/app.js:128760:5
[824]</</ReactCompositeComponentMixin.updateComponent@https://app.augur.net/app.js:128689:7
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[824]</</ReactCompositeComponentMixin.performUpdateIfNecessary@https://app.augur.net/app.js:128637:7
[868]</ReactReconciler.performUpdateIfNecessary@https://app.augur.net/app.js:136013:5
runBatchedUpdates@https://app.augur.net/app.js:136716:5
[892]</</Mixin.perform@https://app.augur.net/app.js:138614:13
[892]</</Mixin.perform@https://app.augur.net/app.js:138614:13
[875]</</<.perform@https://app.augur.net/app.js:136673:12
[875]</</flushBatchedUpdates@https://app.augur.net/app.js:136734:7
[863]</</ReactPerf.measure/wrapper@https://app.augur.net/app.js:135320:16
[892]</</Mixin.closeAll@https://app.augur.net/app.js:138680:11
[892]</</Mixin.perform@https://app.augur.net/app.js:138627:11
[839]</ReactDefaultBatchingStrategy.batchedUpdates@https://app.augur.net/app.js:131337:7
enqueueUpdate@https://app.augur.net/app.js:136763:5
enqueueUpdate@https://app.augur.net/app.js:136351:3
[874]</</ReactUpdateQueue.enqueueSetState@https://app.augur.net/app.js:136517:5
[821]</</ReactComponent.prototype.setState@https://app.augur.net/app.js:128060:3
getStateFromFlux@https://app.augur.net/app.js:49986:1
[179]</StoreWatchMixin/<.componentDidMount/this._setStateFromFlux<@https://app.augur.net/app.js:57965:25
emit@https://app.augur.net/app.js:58133:17
handleUpdateAssets@https://app.augur.net/app.js:54392:5
[178]</Store.prototype.__handleAction__@https://app.augur.net/app.js:57898:7
[174]</Dispatcher.prototype.doDispatchLoop/<@https://app.augur.net/app.js:57608:23
[483]</bindCallback/<@https://app.augur.net/app.js:104815:14
[488]</createBaseFor/<@https://app.augur.net/app.js:104950:11
baseForOwn@https://app.augur.net/app.js:104259:10
[492]</createForOwn/<@https://app.augur.net/app.js:105042:12
[174]</Dispatcher.prototype.doDispatchLoop@https://app.augur.net/app.js:57591:1
[174]</Dispatcher.prototype._dispatch@https://app.augur.net/app.js:57580:5
[174]</defaultDispatchInterceptor@https://app.augur.net/app.js:57534:3
[174]</Dispatcher.prototype.dispatch@https://app.augur.net/app.js:57558:3
[175]</Flux/this.dispatchBinder.dispatch@https://app.augur.net/app.js:57724:9
updateAssets@https://app.augur.net/app.js:47959:7
contracts@https://app.augur.net/app.js:48185:11
sift@https://app.augur.net/app.js:44719:40
[90]</module.exports/poll_contracts_listener/<@https://app.augur.net/app.js:44751:25
[955]</</broadcast/<@https://app.augur.net/app.js:146529:17
[954]</</_once/<@https://app.augur.net/app.js:144775:13
[954]</</async.eachOfSeries/iterate/<@https://app.augur.net/app.js:144987:21
[954]</</only_once/<@https://app.augur.net/app.js:144767:13
[955]</</broadcast/</<@https://app.augur.net/app.js:146522:1
parse@https://app.augur.net/app.js:146175:21
[955]</</post/<@https://app.augur.net/app.js:146318:17
on_end@https://app.augur.net/app.js:147639:5
on_state_change@https://app.augur.net/app.js:147589:7
 app.js:44723:21

0 Object { address: "0x84c9426dfabdd67f5971e01aa670108b3…", topics: Array[4], data: Array[2], blockNumber: "0x7380a", logIndex: "0x0", blockHash: "0x3a0d9c32eb52e767e1707f0ad776da42d…", transactionHash: "0xfad0b6a5d7a7a4ef2782a6821cced878e…", transactionIndex: "0x0" } app.js:44724:21
```

**Comment by tinybike** [comment]
@joeykrug does the `tourMarketKey` error show up every time this happens?  Or did that just happen to appear in the middle of this error?

**Comment by joeykrug** [comment]
Every time

**Comment by tinybike** [comment]
Just to make sure I'm clear on this: **all** that junk appears in the console (including the tourMarketKey contracts filter error) every time?  Trying to figure out if this is one error or a couple errors jumbled together!

**Comment by joeykrug** [comment]
Yes every single time I try to type or delete a letter in the search box

**Comment by tinybike** [comment]
This works in Windows 7/FF 43.0.1 and iOS 9.3/Safari.  Looks like this is a Windows 10-specific bug.

**Comment by joeykrug** [comment]
This bug just stopped occurring.  However there's a new bug with search where it won't show the market even though it clearly found one:

![search](https://cloud.githubusercontent.com/assets/5925937/13376932/6254365c-dd8d-11e5-9de9-d58d8dc130a9.PNG)

**Comment by tinybike** [comment]
Good catch.  Fixed in ef41d6de0dac266be240379bd55d2a918cbb2664.
